modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/DDMFormFieldValueRenderer.java

```diff
@@ -25,4 +26,10 @@
 
 	public String render(DDMFormFieldValue ddmFormFieldValue, Locale locale);
 
+	public default String render(
+		DDMFormFieldValue ddmFormFieldValue, String fieldName, Locale locale) {
```

Comment:
@marcelabc, can we rename `fieldName` to `ddmFormFieldName` to keep consistency with existing back-end code?